### ISSUE TITLE
False positive `unused-private-member` on static members

### ISSUE THREAD

**Issue by EugeneZelenko** [PyCQA/pylint#4682]
Hi!

As of e4cd2ef1b016dece104cccf33bd0d455f335f5af, `unused-private-member` reports false positives for static members used in static methods:

```
class Test():

    __state = False

    @staticmethod
    def method():
        if not Test.__state:
            Test.__state = True
```

**Comment by yushao2** [comment]
This seems like a duplicate of #4681, which was opened by @cdce8p 

cmiiw

**Comment by EugeneZelenko** [comment]
Duplicate of  #4681.
